initial_setup/__init__.py

```diff
@@ -70,7 +68,17 @@ def __init__(self, gui_mode):
         else:
             log.debug("running in TUI mode")
 
-        if external_reconfig:
+        self._external_reconfig = False
+
+        # check if the reconfig mode should be enabled
+        # by checking if at least one of the reconfig
+        # files exist
+        for reconfig_file in RECONFIG_FILES:
+            if os.path.exists(reconfig_file):
+                self.external_reconfig = True
```

Comment:
This could be nicely written as:
``self.external_reconfig = any(os.path.exists(path) for path in RECONFIG_FILES)``
which I personally find better readable, but YMMV.